### PR TITLE
Github Templates - DevKit updates

### PR DESCRIPTION
**Pull request by ElectricMaxxx**

### CODE REVIEW

File: .github/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+about: Report errors and problems
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+**Symfony version(s) affected**: x.y.z
```
Comment (dbu) [inline .github/Bug_report.md]: i am not sure we should have this. we could talk about the library version, but i would just leave this out

File: .github/Documentation_issue.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+name: Documentation Issue
+about: Anything related to Symfony Documentation
```
Comment (dbu) [inline .github/Documentation_issue.md]: Symfony CMF Documentation

File: .github/Documentation_issue.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+Symfony Documentation has its own dedicated repository. Please open your
+documentation-related issue at https://github.com/symfony/symfony-docs/issues
```
Comment (dbu) [inline .github/Documentation_issue.md]: please adjust to the cmf docs repository

File: .github/Security_issue.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+If you have found a security issue in Symfony, please send the details to
+security [at] symfony.com and don't disclose it publicly until we can provide a
```
Comment (dbu) [inline .github/Security_issue.md]: do we have a security contact? i think there is no official one, maybe ask them to email you and me?

File: .github/Security_issue.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+fix for it.
+
+More information: https://symfony.com/security
```
Comment (dbu) [inline .github/Security_issue.md]: we should remove this line, its about the symfony framework not the cmf

File: .github/Support_question.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+We use GitHub issues only to discuss about Symfony bugs and new features. For
+this kind of questions about using Symfony or third-party bundles, please use
+any of the support alternatives shown in https://symfony.com/support
```
Comment (dbu) [inline .github/Support_question.md]: this is also pointing to symfony proper

i would find it ok keep people opening support questions in github - we had it in the past and it was much less volume than what symfony itself would have.

wdyt of adding some guideline similar to bug report to make people explain what they want to do and what the question is?

**Comment by ElectricMaxxx** [comment]
@dbu  better?

**Comment by ElectricMaxxx** [comment]
So i will roll them out on all packages. Thanks.

**Pull request merged**